### PR TITLE
Fix reactivity bug preventing Ollama prompt save

### PR DESCRIPTION
**Pull request by mtsgrd**

### CODE REVIEW

**Review by Caleb-T-Owens — APPROVED** [review]
I spent half an hour trying to figure that out 🤦.

**Comment by krlvi** [comment]
Can someone ELI5 this one to me? Why does this work?

**Comment by Caleb-T-Owens** [comment]
@krlvi 

Under the hood, `foo.bar = 69` roughly equates to `get(foo).bar = 69` which doesn't trigger any subscribers. Adding `foo = foo` introduces something like `foo.set(get(foo))` which then triggers the update.

It seems that with the svelte 5 update, this has gotten less reactive.

**Comment by Caleb-T-Owens** [comment]
AFAIK, updating the property should work, so this might be a svelte related bug.
